stdlib/README.md

```diff
@@ -0,0 +1 @@
+# leo-ast-passes
```

Comment:
We meed some readme there. I can try to write it.

Reply:
I'll write one up today.